api/src/server.ts

```diff
@@ -137,7 +138,7 @@ export async function startServer(): Promise<void> {
 					// No need to log/warn here. The update message is only an informative nice-to-have
 				});
 
-			logger.info(`Server started at http://localhost:${port}`);
+			logger.info(`Server started at http://${host}:${port}`);
```

Comment:
Wondering should we show this as `localhost` when possible, rather than ending up with `Server started at http://0.0.0.0:8055` for example? `0.0.0.0` is definitely technically more accurate, but just not super sure will any users get confused by it.

Checked out the implementation in Vite and seems like they do use a piece of logic for this here: https://github.com/vitejs/vite/blob/b79fec9dd53b91f0629c281448abba573a927e2f/packages/vite/src/node/utils.ts#L583-L611, where they try to make `0.0.0.0`, `::` etc show up as `localhost`.

Just an open question here, not exactly a must-change 😄

Reply:
Hi, 
I understood what you mean but you will lie if you listen on 192.168.1.100, for example, and you'll display http://localhost:8055.
But it seems that vitejs manage it in a good way. If you want to implement it, I will ask you some help because my JS skill is poor. I am an Ops :)

Regards.

Reply:
> if you listen on 192.168.1.100, for example, and you'll display http://localhost:8055/.

That's definitely one of my concern! :)

With that said, I also just recalled Nuxt's approach might also be a good alternative, where if they detect `0.0.0.0`, they'll show the host machine IP instead:

![](https://i.stack.imgur.com/EAkE5.png)

> If you want to implement it, I will ask you some help because my JS skill is poor. I am an Ops :)

No worries. This is just a little thought for discussion. We'll probably merge it as-is, but if there's any changes, we'll take it from here as well 👍 Thanks again